packages/react-dom/server.bun.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+// This file is only used for tests.
+// It lazily loads the implementation so that we get the correct set of host configs.
+
+import ReactVersion from 'shared/ReactVersion';
+export {ReactVersion as version};
+
+export function renderToReadableStream() {
+  return require('./src/server/ReactDOMFizzServerBrowser').renderToReadableStream.apply(
+    this,
+    arguments,
+  );
+}
```

Comment:
cc @sebmarkbage what are our plans for legacy server apis? I would imagine we should support every runtime equally so while we have support for say `renderToString` in Node/Browser we should also have that in Bun, and then when we drop those we will remove them equally from each runtime

Reply:
I've added `renderToString` and `renderToStaticMarkup` to `server.bun.js`.